### PR TITLE
Coinbase redirect addition

### PR DESCRIPTION
**Pull request by benzap**
- added district-ui-mobile dependency

- some minor development environment changes

- updated semantic site.overrides

  - added additional css rule for #mobile-coinbase-overlay

    - Small devices preview: https://i.imgur.com/Q8BY9gf.png

    - Desktop preview: https://i.imgur.com/ZyVDrsI.png

  - also included a tiny fix for the .right-section, which was
    misplacing the logo. The logo is now centered correctly on mobile
    devices.

- added new event :location/nav-to-url, which takes one argument, and
  performs a url redirect.

- :district0x.location/nav-to-url dispatches :location/nav-to-url

- Implementation consists of an position: absolute div anchored to the
  app-bar

- included additional constants for coinbase links. Considering
  placing in district.ui.mobile repo?

- added sub to generate the correct coinbase appstore
  link. Considering placing in district.ui.mobile repo?

### CODE REVIEW

**Comment by benzap** [comment]
fixes #128

File: src/name_bazaar/ui/subs.cljs [hunk 1]
```diff
@@ -96,3 +97,15 @@
       [(subscribe [:registrar.transfer/tx-pending? label])]
       [(subscribe [:ens.set-owner/tx-pending? node])]))
   first)
+
```
Comment (fbielejec) [inline src/name_bazaar/ui/subs.cljs]: remove extra newline

File: src/name_bazaar/ui/constants.cljs [hunk 2]
```diff
@@ -40,3 +40,9 @@
 
 (defn infinite-list-collapsed-item-height [mobile?]
   (if mobile? 70 54))
+
```
Comment (fbielejec) [inline src/name_bazaar/ui/constants.cljs]: extra newline

File: src/name_bazaar/ui/subs.cljs [hunk 3]
```diff
@@ -96,3 +97,15 @@
...
+
+;; TODO: move into district.ui.mobile repo
+(reg-sub
+  ::mobile-subs/coinbase-appstore-link
+  :<- [::mobile-subs/android?]
+  :<- [::mobile-subs/ios?]
+  (fn [[android? ios?]]
+    (cond
+      (true? android?) (:android-mobile-link constants/coinbase)
```
Comment (fbielejec) [inline src/name_bazaar/ui/subs.cljs]: Why not dispatch cond on just the value which is trythy/falsy?

**Comment by fbielejec** [comment]
There are test failures, yet the build is being marked as [passed](https://travis-ci.org/district0x/name-bazaar/builds/423166563#L1203). We've had this problem before, I think it's becasue the tests are wrapped in a [function](https://github.com/district0x/name-bazaar/blob/master/test/server/run_tests.cljs#L19) and return status is the return of that function. you can check that by running the test suite locally with:

```lein doo node "server-tests" once; echo $status```

I suggest running the tests like we do it for [MemeFactory](https://github.com/district0x/memefactory/blob/feature/ui/test/memefactory/tests/runner.cljs):

File: src/name_bazaar/ui/subs.cljs (hunk 3 above)
Comment (benzap) [reply src/name_bazaar/ui/subs.cljs]: true! i'll fix this in the next push

**Comment by benzap** [comment]
I'll fix up the erroneous newlines and the cond within the subscription, but i'll have to be more thorough with the tests as suggested by @fbielejec 

I'll try and get all of this resolved in the next commit

**Comment by benzap** [comment]
So as discussed with @fbielejec, there were failing test cases which involved an older method of checking for gas price changes. A newer and better method will be implemented in a different ticket, since it doesn't have a strong enough relationship to this ticket, which involves frontend work

Related ticket is #131

**Pull request merged**